ouroboros-network/test/Main.hs

```diff
@@ -30,7 +34,9 @@ tests =
   , Test.ChainFragment.tests
   , Test.ChainProducerState.tests
   , Test.Mux.tests
+#ifndef mingw32_HOST_OS
```

Comment:
Would be good to add a comment here reminding us why. Also, in principle it's better to disable based on runtime OS detection than based on CPP, but I'm not going to stand on ceremony.

Reply:
👍 Will change to run time check.